pkg/endpointmanager/conntrack.go

```diff
@@ -79,51 +83,44 @@ func RunGC(e *endpoint.Endpoint, isLocal, isIPv6 bool, filter *ctmap.GCFilter) {
 	}
 }
 
+// skipGC determines whether we should skip running GC for this endpoint.
+func skipGC(e *endpoint.Endpoint) bool {
```

Comment:
Can you change this function to be a endpoint method? (I'm only asking so we can avoid using the endpoint mutex outside of its pkg)